templates/containers/chiselled/index.html

```diff
@@ -0,0 +1,321 @@
+{% extends "containers/chiselled/_base_chiselled.html" %}
+
+{% block title %}Ultra-small Ubuntu-based distroless containers - Chiselled Ubuntu{% endblock %}
+{% block meta_description %}Get started with chiselled Ubuntu: ultra-small Ubuntu container images. Build Distroless
+containers with 6x smaller attack surface and the advantages of a vendor-supported Linux distribution.{% endblock %}
+
+{% block body_class %}is-paper{% endblock body_class %}
+
+{% block content %}
+<section class="p-strip is-shallow">
+  <div class="row--50-50">
+    <div class="col">
+      <h1>Chiselled Ubuntu</h1>
+    </div>
+    <div class="col">
+      <h2>Ultra-small, ultra-secure containerisation</h2>
+      <p>Rethink your containerisation strategy with chiselled Ubuntu—where ultra-small meets ultra-secure. Trim your
+        attack surface by 6x and get the reliability of a vendor-supported Linux distribution.</p>
+      <hr />
+      <a
+        href="https://canonical-public-cloud.readthedocs-hosted.com/en/latest/oci/oci-how-to/create-chiselled-ubuntu-image.html">Learn
+        about using chisel in production&nbsp;&rsaquo;</a>
+      <br />
+      <a href="/containers/contact-us" class="js-invoke-modal">Get support for chiselled Containers&nbsp;&rsaquo;</a>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="u-align--center">
+      <img src="https://assets.ubuntu.com/v1/05f4aaaa-Frame%203363.png" alt="" />
+    </div>
+  </div>
+</section>
+
+<section class="p-strip u-no-padding--top">
+  <div class="row--50-50">
+    <hr class="p-rule" />
+    <div class="col">
+      <p class="p-text--small-caps u-no-margin--bottom u-sv-3">How to reduce container image size</p>
```

Comment:
```suggestion
      <p class="p-text--small-caps u-no-margin--bottom u-sv-3">How to reduce container image size?</p>
```

Reply:
This is according to Figma design, there is a link in the description. I will ask Sophie to apply the correction to copy doc as well.